README.md

```diff
@@ -1,19 +1,34 @@
-# About
+# About #
```

Comment:
The leading hashmarks are not needed. Would you mind if we drop them? I consider this clean and it also reduces the diff.

Keeping the blank line between headings and their body would also be cool. My current take is: on blank between heading and body, two blanks before a headline. Does that work for you?

Reply:
I think we can do without trailing `#`, but adding white lines makes it less readable in raw form, and in rendered form it doesn't make a difference.

Reply:
> but adding white lines makes it less readable in raw form

I don't agree so I guess that's subjective. Normally I don't think a pull request should make style changes like that but I don't think it matters enough to insist on a revert of that part.